README.md

```diff
@@ -200,14 +231,16 @@ operator.
   operator: radiance-operator
   # commands and args will be used both locally and inside the container
   # TODO: This must change to be platform specific
-  command: gensky -c -B '{{ inputs.parameters.desired-irradiance }}' > '{{ inputs.parameters.sky-file }}'
+  command: gensky -c -B {{ inputs.parameters.desired-irradiance }} > file.sky
```

Comment:
From my understanding the way to use them is to encase an entire string which can contain templates in it. In the case above if you wanted to add quotes I would add them like this:

```yaml
  command: "gensky -c -B {{ inputs.parameters.desired-irradiance }} > file.sky"
```

Otherwise it will mess with yaml printing.